nautobot/core/templates/generic/object_retrieve.html

```diff
@@ -163,6 +163,7 @@ <h1>
             <div class="row">
                 <div class="col-md-6">
                     {% include 'inc/object_details_advanced_panel.html' %}
+                    {% block advanced_content_left_page %}{% endblock advanced_content_left_page %}
```

Comment:
Let's document this in `.added` as a new feature for templates to consume if desired. I just searched and it doesn't look like we document the `advanced_content_*` blocks in our existing dev documentation -- it might be good to add to `nautobot/docs/development/core/templates.md` as well though I won't hold you to it.

Reply:
Added.